libminifi/include/core/state/nodes/AgentInformation.h

```diff
@@ -623,18 +632,70 @@ class AgentManifest : public DeviceInformation {
 };
 
 /**
- * Purpose and Justification: Prints classes along with their properties for the current agent.
+ * This class is used for regular heartbeat without manifest
+ * A light weight heartbeat
  */
-class AgentInformation : public DeviceInformation, public AgentMonitor, public AgentIdentifier {
+class AgentInformationWithoutManifest : public DeviceInformation, public AgentMonitor, public AgentIdentifier {
+public:
+
+    AgentInformationWithoutManifest(std::string name, utils::Identifier & uuid)
+    : DeviceInformation(name, uuid) {
```

Comment:
Constructor initializer list should be indented +4 spaces to the right.

Use `std::move(name)` to avoid unnecessary copying of `name`, since `DeviceInformation` takes it by-value, too.